tools/Performance/DynamoPerformanceTests/DynamoPerformanceTests.csproj

```diff
@@ -1,259 +1,50 @@
-      <Link>Properties\AssemblySharedInfo.cs</Link>
-    </Compile>
-    <Compile Include="ResultsComparer.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="Configs\app.config" />
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <Analyzer Include="..\DynamoPerformanceTests\packages\Microsoft.CodeAnalysis.Analyzers.2.6.1\analyzers\dotnet\cs\Microsoft.CodeAnalysis.Analyzers.dll" />
-    <Analyzer Include="..\DynamoPerformanceTests\packages\Microsoft.CodeAnalysis.Analyzers.2.6.1\analyzers\dotnet\cs\Microsoft.CodeAnalysis.CSharp.Analyzers.dll" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <PropertyGroup>
-    <PreBuildEvent>robocopy ..\..\..\..\..\..\bin\AnyCPU\$(Configuration)\ . /s /xd fallback_docs
-if %25ERRORLEVEL%25 GTR 3 ( exit 1 ) else ( exit 0 )</PreBuildEvent>
-  </PropertyGroup>
-</Project>
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <Platforms>AnyCPU;x64;NET60_Windows;NET60_Linux;</Platforms>
+        <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+        <PlatformTarget>x64</PlatformTarget>
+        <TargetFramework Condition="'$(Platform)' == 'AnyCPU' Or '$(Platform)' == 'x64'">net48</TargetFramework>
+        <TargetFramework Condition="'$(Platform)' == 'NET60_Windows' Or '$(Platform)' == 'NET60_Linux'">net6.0</TargetFramework>
+        <OutputType>Exe</OutputType>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <OutputPath Condition=" '$(OutputPath)' == '' ">$(SolutionDir)\bin\$(Platform)\$(Configuration)</OutputPath>
+    </PropertyGroup>
+    
+    <ItemGroup>
+        <Reference Include="DynamoCore">
+            <HintPath>..\..\..\bin\$(Platform)\$(Configuration)\DynamoCore.dll</HintPath>
+        </Reference>
+        <Reference Include="DynamoCoreWpf">
+            <HintPath>..\..\..\bin\$(Platform)\$(Configuration)\DynamoCoreWpf.dll</HintPath>
+        </Reference>
+        <Reference Include="DynamoCoreTests">
+            <HintPath>..\..\..\bin\$(Platform)\$(Configuration)\DynamoCoreTests.dll</HintPath>
+        </Reference>
+        <Reference Include="DynamoCoreWpfTests">
+            <HintPath>..\..\..\bin\$(Platform)\$(Configuration)\DynamoCoreWpfTests.dll</HintPath>
+        </Reference>
+    </ItemGroup>
+    <ItemGroup>
+        <Compile Include="..\..\..\src\AssemblySharedInfoGenerator\AssemblySharedInfo.cs">
+            <Link>Properties\AssemblySharedInfo.cs</Link>
+        </Compile>
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="BenchmarkDotNet" Version="0.11.4" />
+        <PackageReference Include="Microsoft.CodeAnalysis" Version="4.6.0" />
+        <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+        <PackageReference Include="Microsoft.VisualBasic" Version="10.3.0" />
+        <PackageReference Include="NDesk.Options.Patched" Version="0.3.2" />
```

Comment:
could you align this with commandline parser? We've tried to remove ndesk options from the repo.

Reply:
Yes I'll work on that